RUN_SERVER.sh

```
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Agent Orchestrator MCP Server Startup Script
+
+echo "🚀 Starting Agent Orchestrator MCP Server..."
+echo ""
+
+# Load secrets if available
+if [ -f secrets.env ]; then
+    echo "📝 Loading Logfire configuration from secrets.env..."
+    export $(cat secrets.env | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)